test/evergreen/compatibility_test_for_releases.sh

```diff
@@ -693,27 +697,29 @@ if [ "$patch_version" = true ]; then
         # Pick a patch version from the list of patch versions for the release branch
         cd $b; pv=$(pick_a_version $b); cd ..
 
-        (build_branch $pv)
-        rtn=$(is_test_checkpoint_recovery_supported $pv)
-        patch_fix_included=$(git log --oneline --grep=WT-8708 -b "$pv" --)
- 
-        # Only run verify if the picked version supports test checkpoint recovery
-        if [ $rtn == "no" ]; then
-            echo -e "\n\"$pv\" does not support test checkpoint with recovery, skipping ...\n"
-        # Apply patch fix from WT-8708 to already released compatible versions to avoid test/checkpoint setting commit timestamp less than stable timestamp
-        elif [ $rtn == "yes" ] && [ ! $patch_fix_included ]; then
-            cd $pv;
-
-            git format-patch -1 d4b0ad6cacb874fdc20bcc76311d789dd5a01441;
-            patch -p1 < 0001-WT-8708-Fix-timestamp-usage-error-in-test-checkpoint.patch;
-
-            cd ..;
+        if [ -n "$pv" ]; then
```

Comment:
Same question here, is that ok to have a null string here?

Reply:
Same as my previous comment, thanks for explaining @Siddhartha8899, I will resove it!

Reply:
A null string from the `$pv` variable indicates the release branch (e.g. mongodb-6.0) has not yet got a GA version (e.g. 6.0.0, 6.0.1) tagged. It's worth having some comments added here to describe that.

Reply:
I think your comment describes it pretty well. Updated in [49d1e1d](https://github.com/wiredtiger/wiredtiger/pull/7951/commits/49d1e1ddde664b01ef1ed6fcb53867400a19cc88)